packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -416,6 +438,89 @@ contract Deploy is Deployer {
         console.log("SuperchainConfig deployed at %s", address(superchainConfig));
     }
 
+    /// @notice Deploy the DelayedVetoable contract for the SuperchainConfig
+    ///         Note: Because the getters on DelayedVetoable must be read by pranking as the zero
+    ///         address, this function does not use the broadcast modifier, and instead starts and
+    ///         stops broadcasting, then starts and stops pranking, within the function body.
+    function deployDelayedVetoableForSuperchain() public {
+        vm.startBroadcast();
+        SuperchainConfig superchainConfigProxy = SuperchainConfig(mustGetAddress("SuperchainConfigProxy"));
+        DelayedVetoable delayedVetoable = new DelayedVetoable({
+                superchainConfig_: superchainConfigProxy,
+                target_: address(superchainConfigProxy)
+            });
+        vm.stopBroadcast();
+
+        vm.startPrank(address(0));
+        require(delayedVetoable.superchainConfig() == address(superchainConfigProxy));
```

Comment:
Perhaps we could modularize the 2 deployments for delayed vetoable with an internal function to dedupe code